mopidy/config/__init__.py

```diff
@@ -162,32 +161,32 @@ def _load(files, defaults, overrides):
     return raw_config
 
 
-def _load_file(parser, filename):
-    if not os.path.exists(filename):
+def _load_file(parser, file_path):
+    if not file_path.exists():
         logger.debug(
-            'Loading config from %s failed; it does not exist', filename)
+            'Loading config from %s failed; it does not exist', file_path)
```

Comment:
You're right.

For logging/debugging purposes, I think we should be able to figure out the exact bytes in the path, so instead of the replace-decode, I think we should use one of the following approaches:

``` python
In [14]: p = Path(u'/home/jodal/æøå.txt')

In [15]: print u'Loading config from %r failed' % p
Loading config from PosixPath('/home/jodal/\xc3\xa6\xc3\xb8\xc3\xa5.txt') failed

In [16]: print u'Loading config from %r failed' % p.as_uri()
Loading config from 'file:///home/jodal/%C3%A6%C3%B8%C3%A5.txt' failed
```

The latter only works on absolute paths, but all paths we pass through `mopidy.internal.path.expand_path()` are made absolute, so that should generally work out.